### DISCUSSION TITLE
Part of Zigbee2mqtt entities are no longer being provided by the mqtt integration

### DISCUSSION THREAD

**Discussion by cancan92** [community/community#77709] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hello
I'm using Zigbee2mqtt on HA for 4 years and everything was great. This week suddenly I had a mistake, loosing identities coming from Zigbee2mqtt. Ok I have more than 100 zigbee products but it always was ok.
For exemple : I have a switch for my "Bibliothèque". The status is coming right from zigbee but I can't see the identity on lovelace. 
From Zigbee2mqtt I am able to use the switch.
Thus automations and scripts using those entities are no longer working ...

I tried to start from zero with only one zigbee product but it still the same. It looks like switches, lights and thermostats are no more reachable when Xiomi products like temperature, wall switches are ok

Does anyone knows what is my mystake ?

Thanks in advance

<img width="333" alt="Capture d’écran 2023-12-02 à 13 22 46" src="https://github.com/community/community/assets/38877466/3614f2de-5c23-4c76-902b-b619c6a8b41d">
<img width="1026" alt="Capture d’écran 2023-12-02 à 13 38 37" src="https://github.com/community/community/assets/38877466/15e948aa-b082-4415-ba43-89c2ddbac759">

**Comment by cancan92** [comment]
If necessary : I am using Mosquitto broker version: 6.4.0 and last version of Zigbee2mqtt Edge (it is the same trouble with no Edge).
I also tried to delete from HA such an identity no more reachable. It became no more reachable, even when I delete the equipment from the zigbee network and suscribe it again on the zigbee network !
<img width="593" alt="Capture d’écran 2023-12-02 à 14 44 25" src="https://github.com/community/community/assets/38877466/bcebd77c-c7c5-4013-8e46-55b0639cc183">

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

**Reply by cancan92** [reply]
@queenofcorgis : thanks for your answer. unfortunately I'm not asking how to use it but how to solve the issue I met.
I also opened such an issue here : https://github.com/Koenkk/zigbee2mqtt/issues/20004

**Reply by queenofcorgis** [reply]
@cancan92 for use and support of a project hosted on GitHub, I still recommend those links. The issue you opened will be your best avenue to solve it. 

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas.

**Comment by cancan92** [comment, marked as answer]
@queenofcorgis : understood and sorry for my mistake. I close this topic. Thanks again
